core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,26 +2,26 @@
 Benchmark ZStandardCompressionCodec
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_362-b09 on Linux 5.15.0-1035-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 1.8.0_362-b09 on Linux 5.15.0-1036-azure
+Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz
 Benchmark ZStandardCompressionCodec:                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 --------------------------------------------------------------------------------------------------------------------------------------
-Compression 10000 times at level 1 without buffer pool            394            527         151          0.0       39420.6       1.0X
-Compression 10000 times at level 2 without buffer pool            453            455           3          0.0       45251.2       0.9X
-Compression 10000 times at level 3 without buffer pool            640            645           3          0.0       64045.8       0.6X
-Compression 10000 times at level 1 with buffer pool               193            197           3          0.1       19278.3       2.0X
-Compression 10000 times at level 2 with buffer pool               244            251           3          0.0       24416.2       1.6X
-Compression 10000 times at level 3 with buffer pool               420            436          10          0.0       42020.0       0.9X
+Compression 10000 times at level 1 without buffer pool            366            470         137          0.0       36624.7       1.0X
+Compression 10000 times at level 2 without buffer pool            412            413           2          0.0       41169.5       0.9X
+Compression 10000 times at level 3 without buffer pool            538            541           3          0.0       53813.4       0.7X
+Compression 10000 times at level 1 with buffer pool               257            259           1          0.0       25702.7       1.4X
```

Comment:
https://github.com/apache/spark/pull/40721#discussion_r1161989573

It seems that this problem has occurred again. I ran `ZStandardBenchmark` on GA 13 times and did not get the same results as before ... @dongjoon-hyun

Reply:
also cc @pan3793